README.md

```diff
@@ -1310,8 +1311,30 @@ test('passes when using .not.toHaveNetworkAuthenticationRequiredStatus() for 200
     expect(response).not.toHaveNetworkAuthenticationRequiredStatus();
 });
 ```
+
 </details>
 
+<br/>
+
+#### .toHaveHeader(`<header name>`[, `<header value>`])
+
+Use `.toHaveHeader` when checking if response has a specific header, optionally with a specific value
+
+header names are case-insensitive
+
+```js
+test('passes when response header match the expected header', async () => {
+    const response = await axios.get('https://httpstat.us/200');
+    expect(response).toHaveHeader('X-Request-ID');
+});
+
+test('passes when using .toHaveHeader() with expected value', async () => {
+    const response = await axios.get('http://example.com');
+    expect(response).toHaveHeader('Accept', 'application/json');
```

Comment:
This request doesn't have header `accept`, it does however have `content-type` as `text/html; UTF-8`